package.json

```diff
@@ -53,6 +53,7 @@
     "ws": "^7.4.6"
   },
   "devDependencies": {
+    "@angular-devkit/build-angular": "^13.3.7",
```

Comment:
Is there a reason that this needs to move up to the root devDeps? Would prefer we keep platform specific deps scoped to their platforms unless we need to move them at the root

Reply:
Yep, see the PR description....same reason as this PR https://github.com/aws-amplify/amplify-ui/pull/1686

Reply:
I kept it in `amplify-ui-angular-mono`, but moved those from `angular-example` and `angularcli` to root

Reply:
Alright! Thanks for the comment. I just updated the package on the original files 😁